WPFHospitalEditor/HospitalMap.xaml

```diff
@@ -35,8 +35,20 @@
             </ComboBox>
             <Button x:Name="equipmentSearchButton" Content="Search" HorizontalAlignment="Center" VerticalAlignment="Center" Margin="10" Height="30" Width="60" FontWeight="Bold" FontSize="16" Click="Equipment_Search"></Button>
         </StackPanel>
-
+        <StackPanel Height="51" Orientation="Horizontal">
+            <ComboBox x:Name="doctorsComboBox" x:FieldModifier="public" Height="30" Width="200" VerticalContentAlignment="Center" SelectedIndex="0" Margin="20,0,0,0">
+                <ComboBoxItem Name="emptyDoctorComboBox"/>
+            </ComboBox>
+            <TextBox Height="30" Width="200" x:FieldModifier="public" Margin="20,0,0,0" Name="startDateTextBox" ></TextBox>
+            <TextBox Height="30" Width="200" x:FieldModifier="public" Margin="20,0,0,0" Name="endDateTextBox" ></TextBox>
+            <ComboBox x:Name="PriorityComboBox" x:FieldModifier="public" Height="30" Width="120" VerticalContentAlignment="Center" Margin="20,0,0,0" SelectedIndex="0">
+                <ComboBoxItem Content="Doctor"/>
+                <ComboBoxItem Content="Time"/>
+            </ComboBox>
+            <StackPanel x:Name="appointmentSearchStackPanel">
+                <Button x:Name="appointmentSearch" Content="Search" Height="30" Width="60" FontWeight="Bold" FontSize="16" Margin="20,10,0,0" Click="appointmentSearch_Click" />
+            </StackPanel>
```

Comment:
Not sure for now, but this could create problem. It scares me that you could remove button appointmentSearch accidently .

Reply:
Fixed